classes/phing/tasks/ext/inifile/IniFileTask.php

```diff
@@ -330,6 +332,22 @@ public function setHaltonerror($halt)
     }
 
     /**
+     * Set verbose attribute.
+     *
+     * @param string $verbose 'yes', or '1' parsed to true.
+     *
+     * @return void
+     */
+    public function setVerbose($verbose)
+    {
+        $pVerbose = false;
+        if (strtolower($verbose) == 'yes' || $verbose == 1) {
```

Comment:
Maybe the use of `StringHelper::booleanValue($verbose)` would be of interest in here, or what do you think?